infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/util/LinuxInitScriptGenerator.java

```diff
@@ -114,7 +114,7 @@ private String generateNodeStartCommand(String instanceId, String rmUrlToUse, St
         return "nohup " + javaCommand + javaProperties + "  &";
     }
 
-    public String generateDefaultIaasConnectorURL(String DefaultRMHostname) {
+    public static String generateDefaultIaasConnectorURL(String DefaultRMHostname) {
```

Comment:
generating the connector iaas url based on the rm hostname is a bit strange, what if the web port or protocol is not the default (e.g. port 8090, protocol https)?

Reply:
LinuxInitScriptGenerator use `ns.default.prefix.connector.iaas.url` and `ns.default.suffix.connector.iaas.url` in the config file `NodeSource.properties` to generate the default connector iaas url.